README.md

```diff
@@ -467,15 +467,15 @@ function accepts a boolean argument that determines whether deprecated versions
 In the following the changes in the respective AST versions, as well as their current support state,
 are listed.
 
-### 70 (experimental)
+### 70 (current)
```

Comment:
Can you please add a `Supported since 1.0.2 (2019-02-11).` line?

Reply:
Done for 1.0.1 : https://github.com/nikic/php-ast/releases